blocks/table/table.js

```diff
@@ -0,0 +1,11 @@
+export default function decorate(block) {
+  block.querySelectorAll('table tbody tr').forEach((row) => {
+    const cell = row.querySelectorAll('td')[1];
+    const wrapper = document.createElement('p');
+    while (cell.firstChild) {
+      wrapper.append(cell.firstChild);
+    }
+    cell.append(wrapper);
+    cell.classList.add('table-grayed');
```

Comment:
this can be done easier with a query selector: https://www.w3schools.com/CSSref/sel_nth-child.php

Reply:
in CSS or in the `table.js`? can you elaborate?

Reply:
instead of adding the `table-grayed` class to the element, you can just change the css to:

```css
.table td:nth-child(2) {
  color: #768390;
}
```